Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -9,101 +9,72 @@ import Shared
 /// through the `themeManager` variable, while using dependency injection.
 protocol Themeable { }
 
-extension Themeable {
-    var themeSystem: ThemeManager { return ThemeManager.shared }
+extension Themeable {}
```

Comment:
Empty extension?

Reply:
I didn't want to delete it because we will be using it.